cabal.project

```diff
@@ -24,13 +24,26 @@ packages:
          ./plugins/hls-module-name-plugin
          ./plugins/hls-ormolu-plugin
          ./plugins/hls-call-hierarchy-plugin
+         ./plugins/hls-cabal-fmt-plugin
 
 tests: true
 
 package *
   ghc-options: -haddock
   test-show-details: direct
 
+-- stylish haskell
+source-repository-package
```

Comment:
why do you need a custom version of stylish-haskell? maybe it worths a comment

Reply:
i removed it as it does not compile, feel free to readd it with a comment if you need it

Reply:
... and the custom version was needed, as expected :worried: